functions.php

```
@@ -351,3 +351,80 @@ function ($matches) use ($cross_icon) {
+    }
+
+    // Check for wrapper class
+    if (! isset($parsed_block['attrs']['className']) || empty($parsed_block['attrs']['className'])) {
+        return $block_content;
+    }
+
+    $classes = $parsed_block['attrs']['className'];
+
+    // 1. Envira Gallery Wrapper
+    if (strpos($classes, 'envira-gallery-wrapper') !== false) {
+        $gallery_id = 0;
```

PHPMD (2.15.0)

The variable $gallery_id is not named in camelCase. (undefined)

(CamelCaseVariableName)